buddyboss/buddyboss-platform
# Fixed GamiPress badges UI issue on Forum activity

### All Submissions:


Fixes #2649 

### How to test the changes in this Pull Request:

1. Go to Forum
2. Create a new discussion or add a reply to any discussion
3. Check activity on activity page

### Proof Screenshots or Video
https://ibb.co/thTKJxf

### Other information:
